src/containers/RepositoriesContainer/RepositoriesDucks.js

```diff
@@ -0,0 +1,75 @@
+import ApiUtils from '../../utils/ApiUtils.js';
+
+// Actions
+const LOAD = 'react-redux-starter-app/repositories/LOAD';
+const LOAD_SUCCESS = 'react-redux-starter-app/repositories/LOAD_SUCCESS';
+const LOAD_ERROR = 'react-redux-starter-app/repositories/LOAD_ERROR';
+
+const initialState = {
+  isLoading: false,
+  repos: undefined
+};
+
+// Reducers
+const REDUCERS = {
+  [LOAD]: (state, action) => ({
+    ...state,
+    isLoading: true
+  }),
+  [LOAD_SUCCESS]: (state, action) => ({
+    ...state,
+    isLoading: false,
+    repos: action.data.map(repo => ({
+      author: repo.owner.login,
+      avatar: repo.owner.avatar_url,
+      description: repo.description,
+      id: repo.id,
+      name: repo.name,
+      starsCount: repo.stargazers_count,
+      url: repo.html_url
+    }))
```

Comment:
nice transform. A bit in doubt if it should live here or be moved to a separate function.
It reads nicely though.